payments_service/solitude/tests/test_transaction.py

```diff
@@ -0,0 +1,70 @@
+import mock
+from nose.tools import eq_, ok_
+
+from payments_service.base.tests import TestCase
+from payments_service.solitude import constants
+from payments_service.solitude.transaction import Transaction
+from payments_service.auth import SolitudeBuyer
+
+
+class TestTransaction(TestCase):
+
+    def setUp(self):
+        super(self.__class__, self).setUp()
+        self.session = {}
+        self.buyer = SolitudeBuyer('uuid', 'pk')
+        self.solitude.generic.product.get_object_or_404.return_value = {
+            'seller': '/generic/seller/123/',
+            'resource_uri': '/generic/product/456/'
+        }
+        self.solitude.generic.transaction.post.return_value = {
+            'resource_uri': '/generic/transaction/789/',
+            'resource_pk': '789'
+        }
+        self.transaction_mock = mock.Mock(name='transaction.mock')
+        self.solitude.generic.transaction.return_value = self.transaction_mock
```

Comment:
This looks like it's the same as ``setup_transaction()`` in the other class. Maybe there's a way to combine them?